lib/content-services/src/lib/node-comments/services/node-comments.service.ts

```diff
@@ -73,15 +75,25 @@ export class NodeCommentsService implements CommentsService {
     }
 
     private newCommentModel(comment: Comment): CommentModel {
+        const user = new User(comment.createdBy);
+        const userId = user.id?.toString();
```

Comment:
Why are you adding `toString`? As far as I can see Person's id is a string already